gqlgen.yml

```diff
@@ -68,6 +68,8 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.APIFile
   FileDiff:
     model: github.com/evergreen-ci/evergreen/rest/model.FileDiff
+  GenericSubscription:
```

Comment:
I have a slight preference for not using the word generic here since this behaves a bit differently than generic types usually do in OOP. I'd prefer something like GeneralSubscription instead.